tiledb/bioimg/helpers.py

```diff
@@ -32,7 +34,7 @@ def get_or_create(self, name: str, schema: tiledb.ArraySchema) -> Tuple[str, boo
         if name in self.r_group:
             uri = self.r_group[name].uri
         else:
-            uri = os.path.join(self.r_group.uri, name)
+            uri = os.path.join(self._uri, name)
```

Comment:
`os.path.join` will produce platform dependent paths where might be incompatible with the remote storage. An example of that is if a local file on a windows machine ingests a file and saves it in an s3 bucket. Then the array uri contains mixed `/` and `\`